bittensor/config.py

```diff
@@ -62,7 +63,7 @@ class Config(object):
         Crypto.generate_private_ed25519().public_key())
     try:
         __remote_ip_default__ = requests.get('https://api.ipify.org').text
-    except:
+    except RemoteIPException as e:
```

Comment:
Not sure this is going to work, unless you've made requests throw custom exceptions. 
Furthermore, if you handle a specific type of exception,
make sure there is another one that catches the rest.
Finally, log all exception throw before they are handles. This will pay itself back when trying to debug.

Suggestion:
except RemoteIPException as e:
   log(DEBUG, e.msg, e.trace)
        __remote_ip_default__ = 'localhost'
   __
except Exception as e:
 log(DEBUG, e.msg)
 // Do error handling

Reply:
Good point. I added a `raise` to the initial if statement and added a log.